terraform/application/.terraform.lock.hcl

```diff
@@ -6,6 +6,7 @@ provider "registry.terraform.io/eppo/environment" {
   constraints = "1.3.5"
   hashes = [
     "h1:1Af95/IhzW16rbX8kSApfrAi8vwc5+7uVbCeyVaGw2E=",
+    "h1:pceowuRAKcjLd+g4noIJdX6CBIWavlM4BvRTsGfH0uQ=",
```

Comment:
not sure this should be commited?

Reply:
ah yeah it keeps changing this when I `terraform-apply`, not sure why but I'll revert it